demo/data/TreeViewFinder.jsx

```diff
@@ -154,20 +156,20 @@ function fetchInfinitePokemonList() {
 }
 
 function fetchInfinitePokemonTree(nodeId) {
-    IDCounter++;
+    IDCounter += 1;
```

Comment:
did we really want this sort of rules?

I personally prefer the `++` notation.

Reply:
We didn't have any discussion about it, so I kept it